content/doc/developer/views/table-to-div-migration.adoc

```diff
@@ -0,0 +1,88 @@
+---
+title: Table to div layout migration
+layout: developerguide
+---
+
+Jenkins core has changed its form layout from 'table' to 'div' as of Jenkins v2.237.
+
+All core taglib's and views were updated in link:https://github.com/jenkinsci/jenkins/pull/3895[jenkins#3895]
+as part of jira:JENKINS-56109[].
+
+For 99% of plugins they do not require any changes at all as they just use the standard jelly tags
+in Jenkins for their form, such as `f:entry`, `f:textbox`, `f:checkbox`.
+
+== Examples of area which may require changes
+
+Below is some areas that may require you to adjust the form layout or JavaScript in your plugin:
+
+* Adding the td / tr elements directly instead of using the `f:entry` tag, e.g. link:https://github.com/jenkinsci/config-rotator-plugin/pull/2[config-rotator-plugin#2]
+
+* Custom javascript that is brittle to the Jenkins form UI layout, we’ve done our best to introduce shims, to not cause breakage here, 
+but if you have written JavaScript that is reliant on the DOM structure you may need to make adjustments,
+take a look at the JavaScript changes in link:https://github.com/jenkinsci/jenkins/pull/3895[jenkins#3895].
```

Comment:
```suggestion
take a look at the JavaScript link:changes[jenkins#3895].
```

Reply:
this suggestion looks broken how about:
```suggestion
take a look at the JavaScript link:https://github.com/jenkinsci/jenkins/pull/3895[changes in jenkins#3895].
```

Reply:
Clearly I'm unfamiliar with this markup language...